plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergSplitSource.java

```diff
@@ -231,7 +239,11 @@ static boolean fileMatchesPredicate(
         return true;
     }
 
-    private static Domain domainForStatistics(io.trino.spi.type.Type type, Object lowerBound, Object upperBound, boolean containsNulls)
+    private static Domain domainForStatistics(
+            io.trino.spi.type.Type type,
+            @Nullable Object lowerBound,
+            @Nullable Object upperBound,
+            boolean containsNulls)
```

Comment:
nit: preexiting - `mayContainNulls` or `nullsAllowed` would be better name for paramaeter.